tests/unit/test_etcdctl.py

```diff
@@ -103,15 +103,15 @@ def test_etcdctl_command(self):
     def test_etcdctl_environment_with_version_2(self, etcdctl):
         """Validate that environment gets set correctly
         spoiler alert; it shouldn't be set when passing --version"""
-        with patch("etcdctl.check_output") as comock:
+        with patch("etcdctl.sp_run") as comock:
             etcdctl.run("member list", api=2)
             api_version = comock.call_args[1].get("env").get("ETCDCTL_API")
             assert api_version == "2"
 
     def test_etcdctl_environment_with_version_3(self, etcdctl):
         """Validate that environment gets set correctly
         spoiler alert; it shouldn't be set when passing --version"""
-        with patch("etcdctl.check_output") as comock:
+        with patch("etcdctl.sp_run") as comock:
```

Comment:
If you revert back to `check_output` as per previous comment, you'll need to revert here as well.